variables.tf

```diff
@@ -40,6 +40,18 @@ variable "admin_user" {
   default     = "postgres"
 }
 
+variable "resource_proxy" {
+  type = object({
+    cpu     = number,
+    memory  = number
+  })
+  default = {
+    cpu         = 500,
+    memory      = 1024
+    description = "Postgres proxy resources"
+  }
+}
+
```

Comment:
Consider adding validation for the cpu and memory values.
We have done this in Presto: https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/variables.tf#L142-L156

Reply:
Ok, I'll add a section for this.